components/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/APIProvider.java

```diff
@@ -40,7 +40,7 @@
      * @param businessOwnerId ID of the owner.
      * @throws AppManagementException
      */
-    public void deleteBusinessOwner(String businessOwnerId) throws AppManagementException;
+    public boolean deleteBusinessOwner(String businessOwnerId) throws AppManagementException;
```

Comment:
Java doc comments issues and formatting issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files